lib/AssemblyUtil/AssemblyUtilImpl.py

```diff
@@ -71,6 +72,36 @@ def get_assembly_as_fasta(self, ctx, params):
         # return the results
         return [file]
 
+    def get_fastas(self, ctx, params):
+        """
+        Given a reference list of KBase object references constructs a local Fasta file with the sequence data for each ref.
+        :param params: instance of type "GetFASTAParams" -> structure:
+           parameter "ref_lst" of list of String
+        :returns: instance of type "GetFASTAOutput" -> structure: parameter
+           "FASTA" of list of String
+        """
+        # ctx is the context object
+        # return variables are: output
+        #BEGIN get_fastas
+
+        # Param check
+        if not params:
+            raise ValueError("Must provide reference list.")
+
+        ws = Workspace(url=self.ws_url)
+
+        ttf = TypeToFasta(self.callback_url, self.sharedFolder, ws)
+        output = ttf.type_to_fasta(ctx, params)
```

Comment:
You're right. I pass ctx into get_fasta because I need it as a param in assembly_as_fasta, but assembly_as_fasta doesn't use ctx. So I removed all instances in ctx in assembly_as_fasta and removed it as a param. 

fixed the params to just a list of refs typedef list<string> KBaseOjbReferences;

Good advice!